Resources/Locale/en-US/deltav/misc/pda.ftl

```diff
@@ -0,0 +1,4 @@
+ent-HoSPDA = head of security pda
```

Comment:
why is this here

Reply:
idk, it's a change from the Justice Dept PR